bee-message/tests/signature_locked_output.rs

```diff
@@ -0,0 +1,179 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(feature = "cpt2")]
+mod cpt2 {
+    use bee_message::{
+        address::{Address, Ed25519Address},
+        constant::IOTA_SUPPLY,
+        output::{Output, SignatureLockedDustAllowanceOutput, SignatureLockedSingleOutput},
+        Error,
+    };
+    use packable::{bounded::InvalidBoundedU64, error::UnpackError, PackableExt};
+    use std::str::FromStr;
```

Comment:
Import groups

Reply:
Can you explain what you mean?

Reply:
I actually explained it in another comment but it's in the accordion, you need to click "load more"